mllib/src/test/scala/org/apache/spark/ml/classification/LogisticRegressionSuite.scala

```diff
@@ -2006,7 +2075,7 @@ class LogisticRegressionSuite
         assert(prob === Vectors.dense(Array(0.0, 0.0, 0.0, 0.0, 1.0, 0.0)))
         assert(pred === 4.0)
     }
-    // TODO: check num iters is zero when it become available in the model
+    require(modelWithMetadata.summary.totalIterations == 0)
```

Comment:
Also, add `assert(model.summary.totalIterations === 0` after the first of these three tests and add `modelZeroLabel.summary.totalIterations > 0` after the second of these tests.